crates/polars-ops/src/frame/pivot/positioning.rs

```diff
@@ -107,8 +110,10 @@ pub(super) fn position_aggregates_numeric<T: PolarsNumericType>(
     value_agg_phys: &ChunkedArray<T>,
     logical_type: &DataType,
     headers: &StringChunked,
+    default_val: &AnyValue,
```

Comment:
For the numeric variant please accept a numeric value and do the `extract` in the caller.